pkg/controller/pod_volume_backup_controller.go

```diff
@@ -147,47 +144,72 @@ func (r *PodVolumeBackupReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		return ctrl.Result{}, errors.Wrap(err, "error getting backup storage location")
 	}
 
-	// #4820: restrieve insecureSkipTLSVerify from BSL configuration for
-	// AWS plugin. If nothing is return, that means insecureSkipTLSVerify
-	// is not enable for Restic command.
-	skipTLSRet := restic.GetInsecureSkipTLSVerifyFromBSL(backupLocation, log)
-	if len(skipTLSRet) > 0 {
-		resticCmd.ExtraFlags = append(resticCmd.ExtraFlags, skipTLSRet)
+	// name of ResticRepository is generated with prefix volumeNamespace-backupLocation- and end with random characters
+	// it could not retrieve the ResticRepository CR with namespace + name. so first list all CRs with in the volumeNamespace
+	// then filtering the matched CR with prefix volumeNamespace-backupLocation-
+	backupRepos := &velerov1api.BackupRepositoryList{}
```

Comment:
Put the logic to get backupRepository to a dedicate file, so as to remove code redundancy between PVB and PVR?

Reply:
currently, I couldn't find a proper directory, I put it into `pkg/util/crd/backup_repository.go`

Reply:
Personally, I don't see problems with this directory, let's see other reviewers' suggestion.

Reply:
I've moved the `pkg/util/crd/backup_repository.go` into `pkg/repository/util/backup_repo_op.go`